src/tokens.js

```diff
@@ -259,6 +286,30 @@ const tokens = {
     return null;
   },
 
+  /**
+   * Validates if already has a token with same name or symbol added in the wallet
+   *
+   * @param {string} name Token name to search
+   * @param {string} symbol Token symbol to search
+   *
+   * @return {Object|null} Token if name or symbol already exists, else null
+   *
+   * @memberof Tokens
+   * @inner
+   */
+  tokenInfoExists(name, symbol) {
+    const tokens = this.getTokens();
+    for (const token of tokens) {
+      if (token.name === name) {
```

Comment:
Maybe we should do some cleanup, such as removing spaces, etc. Maybe we should prohibit some of them in the CreateTokenTransaction.

Reply:
I've done the strip cleanup, do you think we should do more? Maybe begin with this only?

Reply:
Maybe we should create a cleanup method. I would be concerned about:
1) Uppercase vs Lowercase
2) Spaces at the beginning or at the end
3) Multiples spaces in the middle (in face, should we allow any space at all in the symbol?)
4) Special characters with unexpected results.

Reply:
I've done the cleanup taking care of 1, 2 and 3. Maybe we could discuss more if we should take care of special chars for now. Do you think it's enough for now and I create the task to discuss this point 4?

Reply:
I think it is enough for now. Let's create a new issue and discuss it further before implementing different rules.

Reply:
Ok, perfect